### DISCUSSION TITLE
Seq Okta Integration (logout token)

### DISCUSSION THREAD

**Discussion by specialagentsoftware** [datalust/seq-tickets#1628] [Help]
During our implementation of Seq with Okta we have experienced an issue when you attempt to logout of Seq. When you attempt to logout via Seq you are presented with a client id error from okta. I believe this is due to the fact that Okta requires a token be passed for logout in a parameter named id_token_hint.

https://developer.okta.com/docs/reference/api/oidc/#logout

This does not stop us from using Seq of course. I did want to mention it, as it would allow you to possibly eliminate another blocker / issue for people using Okta.

**Comment by nblumhardt** [comment]
Thanks @specialagentsoftware, we'll take a look.

I found some suggestions out there that Okta supports custom sign-out pages, which you might be able to use as the logout redirect target, and also it seems in some cases a `/login/signout` endpoint is available:

 * https://devforum.okta.com/t/logout-case-sensitive/10690
 * https://devforum.okta.com/t/is-it-possible-to-do-force-logout-without-id-token-hint/20521

Do either of these help?
